src/alfanous-django/templates/base.html

```diff
@@ -281,16 +281,18 @@
             </div><!--/span-->
             </nav>
             <!-- end left navbar -->
-
+            <main>
             <div class="row offset2 span9 text-center">
                 <div class="span7 offset1">
+                  <h1>
                     <a class="logo" href="{% url 'wui.views.results' current.unit %}">
                     {% if LANGUAGE_CODE == "ar" or  LANGUAGE_CODE == "fr" or LANGUAGE_CODE == "id" %}
-                  	  <img src="/static/shared/img/logo_{{ current.unit }}_{{ LANGUAGE_CODE }}.png" alt="logo">
-                  	 {% else %}
-                  	 	 <img src="/static/shared/img/logo_{{ current.unit }}_en.png" alt="logo">
-                  	  {% endif %}
```

Comment:
I think H1 should have a unique value by page, the query for example

Reply:
Most search pages I looked at in the web uses fix one. Could we recheck that after trying that with google. and see reports.

Reply:
@sneetsher no problem, is the PR complete now and ready for merge?

Reply:
just one minor update for sitemap to fix google warning about it's spec. We keep microdata for another separate PR.